pulseapi/entries/views.py

```diff
@@ -26,77 +29,62 @@
 from pulseapi.utility.userpermissions import is_staff_address
 
 @api_view(['PUT'])
+@permission_classes([IsAuthenticated])
```

Comment:
`permission_classes` have nothing to do with `django permission objects`. it just checks whether a user has *permission* to to access that api point. which in this case is by checking if he is authenticated or not. It was a convenience method designed to replace `user.is_authenticated()` flow